0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -71,11 +71,14 @@ For more details on the flow, see the [Interledger Protocol specification](../00
 
 <span class="show alert alert-info">**Note:** Interledger only supports Universal mode as described in the whitepaper. Atomic mode can be used by adjacent subsets of participants in an Interledger payment if desired, but this is not part of the standard.</span>
 
-### Error Recovery and Connector Risk
+### Connector Risk and Mitigation
 
-An ILP execution chain can be interrupted if a connector fails to deliver the condition fulfillment. In this case, the sender thinks the transaction failed and retries it using the same condition as the original transaction. If the retry goes through the same connector that failed the first time, that connector can claim the incoming funds without sending a second outgoing transfer.
+Interledger protects senders from risk but connectors accept some risk in exchange for the revenue they generate from facilitating payments. In the Interledger payment flow, connectors' outgoing transfers are executed before their incoming transfers. Once each connector is notified that the outgoing transfer has been executed, they have a window of time to deliver the fulfillment and execute the incoming transfer. Connectors that fail to deliver the fulfillment in time may lose money.
+
+If some transfers in an Interledger payment are executed and others expire, the receiver will think the payment was completed but the sender may think the whole payment failed. Senders may retry payments that expire and they should use the same condition as the original payment. If the second attempt takes the same route, the connector that failed the first time can complete the payment by submitting the fulfillment without sending another outgoing transfer. Connectors should prefer senders (or aggregators of senders such as wallet services) and connectors that retry payments through the same route, because they increase the likelihood of completing such payments. Connectors may incentivize this behavior by offering better rates to parties known to retry failed payments through the same route.
```

Comment:
I think that's overkill for the architecture overview. The point of this paragraph is to say that retries are possible, helpful, and may be incentivized. In any case, connectors should use all of the mitigations included in the other RFC to avoid having this happen. In some ways, why the sender doesn't retry is irrelevant because the connector loses money anyway (whether it was an attack, the sender heard the receiver got the funds out of band, or something else)